apps/ledger-live-desktop/src/newArch/features/ModularDrawer/hooks/useDetailedAccounts.tsx

```diff
@@ -52,11 +53,23 @@ export const useDetailedAccounts = (
 
   const accounts = useMemo(() => {
     const accountTuples = getAccountTuplesForCurrency(asset, nestedAccounts, accountIds);
-    return accountTuples.sort((a, b) => sortAccountsByBalance(a.account, b.account));
+    return orderBy(accountTuples, [(tuple: AccountTuple) => tuple.account.balance], ["desc"]);
   }, [asset, nestedAccounts, accountIds]);
 
+  const overridedAccountName = useBatchMaybeAccountName(accounts.map(({ account }) => account));
```

Comment:
thinking out loud here but I'm wondering how feasible it would be to instead use `getAccountName` that's used by `useBatchMaybeAccountName`, and call it directly within the accounts map below instead 
it sounds a bit complex but it just doesn't feel _right_ to create this array and then reference it by index